ROX-28825: Add ACSCS Central access restriction documentation (#92938)
openshift/openshift-docs: cloud_service/rhacs-cloud-service-service-description.adoc

To prevent information leakage, logs are shipped to SRE through our security information and event management (SIEM) application, Splunk.

[id="access-restriction_{context}"]
=== Central access restriction by IP addresses and CIDR ranges

> Restricting access to Central ensures that only trusted IP addresses can interact with the security policy enforcement point, thereby strengthening the overall security posture.

Reviewer: ```suggestion
Restricting access to Central ensures that only traffic from trusted IP addresses can reach the security policy enforcement point, thereby strengthening the overall security posture.
```

Reviewer: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism
